src/Themes/Custom/assets/webpack/scss/base/_type.scss

```diff
@@ -0,0 +1,5 @@
+p {
+  margin: 0 0 20px;
```

Comment:
Is het slim om met px te werken? Niet beter met `em`? Of een waarde van een bootstrap variabele te gebruiken?

Reply:
Weggehaald, we gebruiken standaard bootstrap  idd, kwam uit Fork theme die gekopieerd was.

Reply:
Er zijn er nog hoor. Misschien eens zoeken naar px in de map `src/Themes/Custom/assets/webpack/scss`?